jupyter-extension/cypress/integration/repo-datum-mode-switch.ts

```diff
@@ -0,0 +1,60 @@
+describe('switching between repo and datum mode', () => {
+  beforeEach(() => {
+    cy.unmountAllRepos();
+    cy.resetApp();
+    cy.isAppReady();
+    cy.openMountPlugin();
+    cy.findAllByText('Mount');
+  });
+
+  it('should open datum mode', () => {
+    cy.findByTestId('Datum__mode').click();
+    cy.findAllByText('Test Datums').should('have.length', 1);
+    cy.findByTestId('Datum__inputSpecInput')
+      .invoke('attr', 'placeholder')
+      .should('contain', 'repo: images');
+  });
+
+  it('mounted repos appear in datum input spec and again when switching back', () => {
+    cy.findAllByText('Mount').first().click();
+    cy.findByTestId('ListItem__select').select('branch');
+    cy.findAllByText('Mount').first().click();
+    cy.findAllByText('Unmount').should('have.length', 2);
+
+    cy.findByTestId('Datum__mode').click();
+    cy.findByTestId('Datum__inputSpecInput')
+      .invoke('prop', 'value')
+      .should('contain', 'cross:');
+    cy.findByTestId('Datum__back').click();
+
+    cy.wait(3000);
+    cy.findAllByText('Unmount').should('have.length', 2);
+    cy.findAllByText('images').first().click();
+    cy.findAllByText('liberty.png').should('have.length', 1);
+    cy.findAllByText('images_branch').first().click();
+    cy.findAllByText('branch.png').should('have.length', 1);
+  });
+
+  it('modifying input spec saves and restores it when back in datum mode', () => {
+    cy.findByTestId('ListItem__select').select('branch');
+    cy.findAllByText('Mount').first().click();
+    cy.findAllByText('Unmount').should('have.length', 1);
+
+    cy.findByTestId('Datum__mode').click();
+    cy.findByTestId('Datum__inputSpecInput')
+      .invoke('prop', 'value')
+      .should('contain', 'name: images_branch');
+    cy.findByTestId('Datum__inputSpecInput')
+      .clear()
+      .type('abcd')
```

Comment:
@pleeko Another testing issue here too. Have you seen this before too? It doesn't finish typing the string into the input spec. https://dashboard.cypress.io/projects/dttaoj/runs/67fe6562-1dd8-4d7c-9f8d-f859bfb97841/test-results/feae4052-db72-46c1-93c0-b58924f50a85

Reply:
Looks like you can add a `delay` or `timeout` parameter to type. Have you tried that?